### PR TITLE
{bio}[dummy] FoldX 3.0 beta 6.1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
They renamed the binary, but didn't rename the 'source' tarball. Great.

### CODE REVIEW

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyconfigs/f/FoldX/FoldX-3.0-beta6.eb [hunk 1]
```diff
@@ -9,6 +9,7 @@ toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 # no source URLs because registration is required to obtain sources
 sources = ['%(name)s_30b6_linux64.zip']
+checksums = ['d3264460a4fe08b57553ebcdaad2dcad']
```
Comment (wpoely86) [inline easybuild/easyconfigs/f/FoldX/FoldX-3.0-beta6.eb]: maybe add something about the not renaming?

Comment (boegel) [inline easybuild/easyconfigs/f/FoldX/FoldX-3.0-beta6.eb]: OK, will do

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8d602a0a1a3fc924164a for a full test report.

**Pull request merged**